src/fhirtypes/ElementDefinition.ts

```diff
@@ -1266,11 +1302,9 @@ export class ElementDefinition {
     delete slice.slicing;
     slice.id = `${this.id}:${name}`;
     slice.sliceName = name;
-    // When a choice is sliced, we do not inherit min cardinality, but rather make it 0
+    // When a we slice, we do not inherit min cardinality, but rather make it 0
```

Comment:
There's a straggling `a` that got left behind in the edit.  I also think it might be good to also briefly note that this allows multiple slices to be defined without violating the sliced element's min and/or that we expect the card will be explicitly constrained after the slice is created.